tests/api/clever/test_clever.py

```diff
@@ -89,190 +93,238 @@ def mock_integration(self):
         integration.setting(MockAPI.OAUTH_TOKEN_EXPIRATION_DAYS).value = 20
         return integration
```

Comment:
I think it was wrong for the original code to make this a `@property`; accessing it has a side effect (creates a new external integration every time). I suspect someone wrote this not realizing the original method had a side effect. The new fixtures make it clearer when something has a side effect.

Reply:
Removed `@property` https://github.com/ThePalaceProject/circulation/pull/825/commits/a537f274feadb80e1fa9336a1f313e7f2a35b5a6